### PR TITLE
[nl]: Add missing Dutch translations

### PR DESCRIPTION
**Pull request by madman-81**
Add missing Dutch translations

### CODE REVIEW

File: locales/nl/json.json [hunk 1]
```diff
@@ -459,10 +459,10 @@
...
+    "Please enter your password to confirm you would like to logout of your other browser sessions across all of your devices.": "Voer uw wachtwoord in om te bevestigen dat u zich wilt afmelden bij uw andere browsersessies op al uw apparaten.",
     "Please provide a maximum of three receipt emails addresses.": "Geef een maximum van drie factuur e-mailadressen.",
     "Please provide the email address of the person you would like to add to this team.": "Geef het e-mailadres op van de persoon die je aan dit team wilt toevoegen.",
-    "Please provide the email address of the person you would like to add to this team. The email address must be associated with an existing account.": "Please provide the email address of the person you would like to add to this team. The email address must be associated with an existing account.",
+    "Please provide the email address of the person you would like to add to this team. The email address must be associated with an existing account.": "Geef het e-mailadres op van de persoon die je aan dit team wilt toevoegen. Het e-mailadres moet gekoppeld zijn aan een bestaand account",
```
Comment (andrey-helldar) [inline locales/nl/json.json:465]:
```suggestion
    "Please provide the email address of the person you would like to add to this team. The email address must be associated with an existing account.": "Geef het e-mailadres op van de persoon die je aan dit team wilt toevoegen. Het e-mailadres moet gekoppeld zijn aan een bestaand account.",
```

**Review by andrey-helldar — APPROVED** [review]

**Comment by andrey-helldar** [comment]
It's ok for me.

**Comment by andrey-helldar** [comment]
@WhereIsLucas, can you check this PR?

**Review by WhereIsLucas — APPROVED** [review]
Ok for me

**Comment by andrey-helldar** [comment]
Dankjewel

**Pull request merged**